components/org.wso2.carbon.identity.api.user.push.device.handler.v1/src/main/resources/push-auth.yaml

```diff
@@ -0,0 +1,780 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DeviceDTO'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+        500:
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+
+      tags:
+        - me
+
+  /me/push-auth/devices/{deviceId}/remove:
+    post:
+      description: |
+        This API is used to remove a device via the mobile app.<br/>
+        <b>Permission required:</b>
+         * /permission/admin/manage/identity/user/push_divice_mgt/delete <br/>
+          <b>OAuth2 Scopes:</b>
+         * internal_identity_mgt_delete
+      summary: |
+        Remove device from mobile app.
+      parameters:
+        - name: deviceId
+          in: path
+          description: Unique Id of the device
+          required: true
+          schema:
+            type: string
+            example: "b03f90c9-6723-48f6-863b-a35f1ac77f57"
+      requestBody:
+        description: 'Account details sent by mobile application'
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RemoveRequestDTO"
+      responses:
+        200:
+          description: Device was removed
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StatusDTO"
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+        404:
+          description: Not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+        500:
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+      tags:
+        - me
+
+
+  #admin apis
+  /{user-id}/push-auth/devices/{deviceId}:
+    delete:
```

Comment:
Lets reorder to have post, get, put, delete. Check the other endpoints and do the same

Reply:
Updated